config.js

```diff
@@ -8,6 +8,7 @@ let config = {
     inactive_user_days : process.env.INACTIVE_USER_DAYS || 60,
     remind_application_days: process.env.REMIND_APPLICATION_DAYS || 30,
     inactive_application_days : process.env.INACTIVE_APPLICATION_DAYS || 45,
+    remind_submission_days: process.env.INACTIVE_SUBMISSION_DAYS || 60,
```

Comment:
Can we change the env var to **INACTIVE_SUBMISSION_NOTIFY_DAYS**, we'll add a new one for deletion days in another ticket.

Reply:
changed applied.